Assets/Scripts/CollectiblesScript.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CollectiblesScript : MonoBehaviour {
+    public int optionalNumber = 0;
+    private float rotateSpeed = 0.3f;
+
+	void Start ()
+    {
+		
+	}
+	
+	void Update ()
+    {
+        transform.RotateAround(transform.position, Vector3.up, rotateSpeed);
+	}
+
+    private void OnTriggerEnter(Collision collision)
+    {
+        Destroy(this.gameObject);
+    }
```

Comment:
Does this work? `OnTriggerEnter` should take in a `Collider` https://docs.unity3d.com/ScriptReference/Collider.OnTriggerEnter.html